libxw/setup.cfg

```diff
@@ -0,0 +1,58 @@
+# https://setuptools.pypa.io/en/latest/userguide/declarative_config.html
+
+[metadata]
+name = libxw
+version = 0.0.1
+description = Basic crossword data structures and algorithms.
+long_description = file: README.md
+maintainer = Kevin Murphy
+maintainer_email = kevin@murp.us
+url = https://github.com/arobey1/crosswords
+license = MIT
+license_files =
+  ../LICENSE
+
+# https://pypi.org/classifiers/
+classifiers =
+  Development Status :: 1 - Planning
+  Intended Audience :: Developers
+  License :: OSI Approved :: MIT License
+  Programming Language :: Python
+  Programming Language :: Python :: 3
+  Programming Language :: Python :: 3 :: Only
+  Programming Language :: Python :: 3.10
+
+[options]
+# Keep this in sync with `$_XW_PYTHON` (in "setup-env").
+python_requires =
+  == 3.10.*
+
+# These are the "abstract dependencies".  The corresponding "concrete dependencies"
+# should be specified in "requirements.txt".
+#
+# https://packaging.python.org/discussions/install-requires-vs-requirements/
+# https://caremad.io/posts/2013/07/setup-vs-requirement/
+install_requires =
+
+# https://setuptools.readthedocs.io/en/latest/userguide/miscellaneous.html#setting-the-zip-safe-flag
+zip_safe = no
+
+package_dir =
+  = src
+packages = find:
+
+[options.packages.find]
+where = src
```

Comment:
This weirdness is basically telling `setuptools` where our actual source code lives, relative to the project root (the parent of this file).  We're nesting inside a `src/` directory here [because that's what the PyPA recommends](https://packaging.python.org/en/latest/tutorials/packaging-projects/#packaging-python-projects) these days.